docs/developers-guide/driver-changelog.md

```diff
@@ -26,16 +30,11 @@ title: Driver interface changelog
   `metabase.driver.sql.query-processor/honey-sql-version` is now deprecated and no longer called. All drivers are
   assumed to use Honey SQL 2.
 
-- The method `metabase.driver.sql-jdbc.sync.interface/active-tables` that we added in 47 has been updated to require
-  an additional argument: `database`.
-  The new function arglist is `[driver database connection schema-inclusion-filters schema-exclusion-filters]`.
-
```

Comment:
I'm not sure if deleting this is the right thing to do, or whether we should leave it on record that we vacillated on this change.

Reply:
Deleting it is fine imo, nobody should be depending on unreleased changelogs because they're subject to change before the release.